components/search/SearchForm.vue

```diff
@@ -284,7 +194,7 @@
       suggestionLinkGen(suggestion) {
         const query = {
           view: this.view,
-          query: `"${suggestion}"`
+          query: `${suggestion}`
```

Comment:
This still needs to be escaped, otherwise any of the suggestions with parentheses won't return results.
For example type 'artist' and select one of the options that contain '[NAME] (artist)'

Reply:
I hadn't realised that, I see your point. I have added quotes back in to escape suggestions.